src/plugins/template/template.c

```diff
@@ -69,6 +69,11 @@ int elektraTemplateError (Plugin * handle ELEKTRA_UNUSED, KeySet * returned ELEK
 	return ELEKTRA_PLUGIN_STATUS_SUCCESS;
 }
 
+int elektraTemplateCommit (Plugin * handle ELEKTRA_UNUSED, KeySet * returned ELEKTRA_UNUSED, Key * parentKey ELEKTRA_UNUSED)
+{
+	return elektraTemplateSet (handle, returned, parentKey);
```

Comment:
We should not recommend to do this. It is better to keep set and commit logic separated. (Only for migration purposes it is fine.)